org.adempiere.base/src/org/compiere/model/MRMA.java

```diff
@@ -47,7 +47,7 @@ public class MRMA extends X_M_RMA implements DocAction
 	/**
 	 * 
 	 */
-	private static final long serialVersionUID = 6196067840638153414L;
+	private static final long serialVersionUID = -8352164928046804628L;

```

Comment:
The line 173 of MRMA differs from the master commit:

in the master commit is get_TrxName()

in this commit is left using no transaction (null)

is that OK?

Reply:
Done, please check. Thanks.